HeterogeneousCore/CUDATest/plugins/TestAlgo.h

```diff
@@ -0,0 +1,13 @@
+#ifndef HeterogeneousCore_CUDATest_plugins_TestAlgo_h
+#define HeterogeneousCore_CUDATest_plugins_TestAlgo_h
+
+#include "CUDADataFormats/PortableTestObjects/interface/TestDeviceCollection.h"
+#include "CUDADataFormats/PortableTestObjects/interface/TestHostCollection.h"
+
+class TestAlgo {
```

Comment:
`TestAlgo` sounds quite generic name, how about placing it in e.g. `cudatest` namespace?

Reply:
OK.
What about the `TestPortableProducerCPU` and the other ED modules ?

Reply:
In principle placing them in a namespace too would be better. I felt the name to be so specific that the probability for clashes in the global namespace would be minuscule. Also the other modules in `HeterogeneousCore/CUDATest/plugins/` are not in a namespace. For those reasons I didn't suggest it.

Reply:
OK, done !